CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this
 project adheres to [Semantic Versioning](http://semver.org/).
 
-<!-- ## Unreleased -->
+## Unreleased
+
+- [**BREAKING**] Benchmark paths are now interpreted relative to the location of
```

Comment:
Non actionable. Worth adding a migration comment here for impacted users. Maybe refer to the `root` option?

Reply:
Added an example migration to the changelog